inst/@sym/mpower.m

```diff
@@ -37,45 +37,18 @@
     return
   end
 
+    cmd = { 'x, y = _ins'
+            'if x.is_Matrix:'
+            '    return sympy.MatPow(x, y) + zeros(x.rows),'
```

Comment:
This is causing a doctest failure.  `sympy.MatPow(x, y).doit()` might be better...